docs/tutorials/01-fee/05-scaffold-react.md

```diff
@@ -0,0 +1,156 @@
+- name: bo
+  coins: 
+  - 20000token
+  - 200000000stake
+  mnemonic: uphold tube excess primary night armor circle puzzle pet memory empower conduct blush hat glare tortoise into embody powder raise punch promote kidney catalog
+- name: charlie
+  coins: 
+  - 20000token
+  - 200000000stake
+  mnemonic: rug cotton ceiling olive cake october way spy million grain actress sponsor
+- name: damian
+  coins: 
+  - 20000token
+  - 200000000stake
+  mnemonic: mansion wing limit daughter allow fiscal attend planet viable giggle prison ready
+# plus-diff-end
+client:
+  openapi:
+    path: docs/static/openapi.yml
+faucet:
+  name: bob
+  coins:
+  - 5token
+  - 100000stake
+validators:
+- name: alice
+  bonded: 100000000stake
+```
+
+## Scaffold a React App
+
+Scaffold a React app using the following command:
+
+<CodeBlock className="language-bash" title=<a href="https://github.com/srdtrk/cosmoverse2023-ibc-fee-demo/commit/51b8e3304f5c8c2e2e477dac9985237be676254b">View Source</a>>
+ignite scaffold react
+</CodeBlock>
+
+For this will create a React app within the `react/` directory. The app depends on a typescript client that is yet to be generated.
+The following command generates the typescript client for the chain, including the Fee Middleware types, and its react hooks.
+
+<CodeBlock className="language-bash" title=<a href="https://github.com/srdtrk/cosmoverse2023-ibc-fee-demo/commit/6b31da02d1ca4fba70d58a3b3e56d9f1e3f06833">View Source</a>>
+ignite generate hooks --clear-cache
+</CodeBlock>
+
+:::caution
+
+You need to add `--clear-cache` to the command above to make sure that the custom modules' (such as Fee Middleware) types are generated.
+
+:::
+
+### Update the Dependencies
+
+The generated React app and ts-client depends on the `@cosmjs` packages. We need to update the dependencies to the latest version because the Cosmos SDK version is not fully compatible with the version of `@cosmjs` packages used by the generated app.
+Run the following commands in the root directory of the project to update the dependencies.
+
+<CodeBlock className="language-bash" title=<a href="https://github.com/srdtrk/cosmoverse2023-ibc-fee-demo/commit/dffcf719fd8c223be06d3327e6bc24ec7cf0cec3">View Source</a>>
+cd ts-client
+npm install @cosmjs/launchpad@0.27.1 @cosmjs/proto-signing@0.31.1 @cosmjs/stargate@0.31.1 @keplr-wallet/types@0.11.14
+</CodeBlock>
+
+<CodeBlock className="language-bash" title=<a href="https://github.com/srdtrk/cosmoverse2023-ibc-fee-demo/commit/1c90ffddbab1655038dc296874f427b036afb749">View Source</a>>
+cd react
+npm install @cosmjs/proto-signing@0.31.1 @cosmjs/stargate@0.31.1 @cosmjs/encoding@0.31.1
+</CodeBlock>
+
+### Fix the Bugs
+
+There are a bug in the generated app that we need to fix. While this is fixed in the next version of Ignite CLI, we need to fix it manually for now since we are using the latest version of Ignite CLI. (The next version of ignite comes with the Fee Middleware wired!)
```

Comment:
```suggestion
There is a bug in the generated app that we need to fix. While this is fixed in the next version of Ignite CLI, we need to fix it manually for now since we are using the latest version of Ignite CLI. (The next version of ignite comes with the Fee Middleware wired!)
```